pages/src/TodoList.js

```diff
@@ -5,8 +5,13 @@ import { createSelector } from 'reselect'
 import { injectState, provideState, update, mergeIntoState } from "freactal";
 import { Map } from 'immutable'
 
+const persistEventInputValue = fn => (event, ...args) => fn(event.target.value, ...args)
+const persistEvent = fn => (event, ...args) => {
+  event.preventDefault()
+  return fn(event, ...args)
```

Comment:
Do not pass the event to the effect, it cannot be used anyway (because not persisted).

Reply:
I taught that effects will inject in the child this way but I was wrong :)